addons/account/models/account_move_line.py

```diff
@@ -415,6 +415,7 @@ class AccountMoveLine(models.Model):
              "associated partner",
     )
     is_refund = fields.Boolean(compute='_compute_is_refund')
+    is_downpayment = fields.Boolean()
```

Comment:
Seems weird to add a stored field on all the move lines just for that. Is there no other way ? Can't it be computed ? What kind of processing do you apply with that field after the invoice has been created ? :thinking:

Reply:
We already are storing this field on all account move lines like this as soon as you install the sale module

Reply:
I'm not really sure how we would be able to guess it from the account and balance sign as it's just on the Product Sales account? The display_type is currently set to product, I could add a display_type of down_payment but I'm not sure if this is a better solution than the current boolean as we would need to change a lot of the checks of type 'display_type == product' to also include lines with display type of down_payment. (but maybe I'm missing something?)

Reply:
The move lines table is huge (a medium-sized accounting contains millions of records in that one) ; that's why I'd prefer to avoid adding one field here :thinking: But for now, I suppose it's not much worse than it used to be (since this field was in sale anyway). So let's keep it this way. But it keep in mind if would be nice to remove it in the future if we can.